release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -587,6 +587,41 @@ For more information, see xref:../operators/operator_sdk/java/osdk-java-quicksta
 [id="ocp-4-11-machine-api"]
 === Machine API
 
+[id="ocp-4-11-mapi-aws-imdsv2"]
+==== Configuration options for the Amazon EC2 Instance Metadata Service
+
+You can now use machine sets to create compute machines that use a specific version of the Amazon EC2 Instance Metadata Service (IMDS). Machine sets can create compute machines that allow the use of both IMDSv1 and IMDSv2 or compute machines that require the use of IMDSv2.
+
+For more information, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-imds-options_creating-machineset-aws[Machine set options for the Amazon EC2 Instance Metadata Service].
+
+[id="ocp-4-11-mapi-ultra-disks"]
+==== Machine API support for Azure ultra disks
+
+You can now create a machine set running on Azure that deploys machines with ultra disks. You can either deploy machines with ultra disks as data disks, or deploy machines with ultra disks by using persistent volume claims (PVCs) that use in-tree or Container Storage Interface (CSI) PVCs.
```

Comment:
Suggestion>> _You can either deploy machines with ultra disks as data disks, or by using persistent volume claims (PVCs) that use in-tree or Container Storage Interface (CSI) PVCs._